nengo/probe.py

```diff
@@ -28,9 +30,10 @@ def validate(self, probe, attr):
                                 "into 'weights'. Probe 'weights' instead.",
                                 since="v2.1.0")
         if attr not in probe.obj.probeable:
-            raise ValidationError("Attribute %r is not probeable on %s.\n"
+            raise ValidationError("Attribute %r is not probeable on %s.%s"
```

Comment:
Hm... I think the `\n` is preferable to `os.linesep` here... this is going to be printed to the terminal, and I don't think any terminal protocols require `\r`; if it's going to be printed to a file, Python automatically deals with the different line endings. From what I can google, it seems like `os.linesep` is only useful when reading/writing from files in bytes mode rather than ascii. We also don't do this in plenty of other cases, so even if it's wrong I would rather switch them all than just have this one switched.